plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -237,10 +288,38 @@ class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
         }
       }
     }
+    if (skipRedundantAnimator(targets, cameraAnimator.type)) {
+      if (debugMode) {
+        logI(
+          TAG,
+          "Animation ${cameraAnimator.type.name}(${cameraAnimator.hashCode()}) was skipped."
+        )
+      }
+      return false
+    }
     cameraAnimator.setObjectValues(*targets)
     return true
   }
 
+  private fun skipRedundantAnimator(targets: Array<out Any?>, type: CameraAnimatorType): Boolean {
+    if (targets.size <= 1) {
+      return true
+    }
+    for (i in 0 until targets.size - 1) {
+      if (targets[i] != targets[i + 1]) {
+        return false
+      }
+    }
+    return when (type) {
+      CameraAnimatorType.CENTER -> Objects.equals(currentCameraState?.center, targets[0])
+      CameraAnimatorType.ZOOM -> Objects.equals(currentCameraState?.zoom, targets[0])
+      CameraAnimatorType.ANCHOR -> false
```

Comment:
could you add some comments why we returns false for ANCHOR here(skip anchor) but in `private fun skipNativeSetCamera(cameraOptions: CameraOptions): Boolean` we returned true if ANCHOR exists?

Reply:
shall we provide the context in the inline comment?

Reply:
emm. strange, I remember 
```
    if (cameraOptions.anchor != null) {
      return false
    }
```
was returning true before, never mind.